packages/tezos-stdlib-unix/tezos-stdlib-unix.9.0/opam

```diff
@@ -13,7 +13,10 @@ depends: [
   "ptime" { >= "0.8.4" }
   "mtime" { >= "1.0.0" }
   "conf-libev"
+  "domain-name" { < "0.3.1" }
   "ipaddr" { >= "4.0.0" }
+  "ezjsonm" { >= "1.1.0" }
+  "fmt" { >= "0.8.7" }
```

Comment:
Could you move these in the `conflicts` section? As I understand it they’re not explicit dependencies and only used implicitly through other libraries via domain-name

Reply:
Done

Reply:
Same for `domain-name`

Reply:
Right...